webpack/global_index.js

```diff
@@ -9,3 +11,16 @@ registerReducer('puppet', reducers);
 registerFills();
 // TODO: the checkForUnavailablePuppetclasses is very nasty
 registerLegacy();
+
+// register columns for React hosts index page
+const puppetHostsIndexColumns = [
+  {
+    columnName: 'environment',
```

Comment:
```suggestion
    columnName: 'puppet_environment',
```

Just to make sure it does not get confused with lifecycle_environment in Katello.

Reply:
Would be nice, but

1) The existing column name for the legacy page is already `environment` and I need to keep it the same, otherwise users' table preferences won't carry over to the new page.
2) the Katello column is `lifecycle_environment` so in this case we won't have a conflict.

Reply:
I see. I guess that is a leftover from the time that puppet was still part of core.